### ISSUE TITLE
Missing commands prevents rendering a menu

### ISSUE THREAD

**Issue by akosyakov** [theia-ide/theia#827]
I've got the following error:
```
Uncaught (in promise) Error: A command with id 'core.save' does not exist.
    at ActionMenuNode.get [as label] (http://localhost:3000/bundle.js:36121:23)
    at ActionMenuNode.get [as sortString] (http://localhost:3000/bundle.js:36137:46)
    at http://localhost:3000/bundle.js:36066:20
    at Array.sort (native)
    at CompositeMenuNode.addNode (http://localhost:3000/bundle.js:36065:24)
    at MenuModelRegistry.registerMenuAction (http://localhost:3000/bundle.js:35981:23)
    at CommonFrontendContribution.registerMenus (http://localhost:3000/bundle.js:23101:18)
    at MenuModelRegistry.onStart (http://localhost:3000/bundle.js:35966:25)
    at FrontendApplication.<anonymous> (http://localhost:3000/bundle.js:18058:36)
    at step (http://localhost:3000/bundle.js:17937:23)
```

We should handle it gracefully, report to a user and ensure that a menu item either not rendered or rendered as disabled.

**Comment by theBadT** [comment]
@akosyakov Hi, I'd like to have a look into the issue. Is there any more input available? like a scenario for reproduction?

**Comment by akosyakov** [comment]
You can build an extension which injects menu and command registries and register a menu before a command.

**Comment by vince-fugnitto** [comment]
Closed thanks to #5894
